test_util/installer_api_test.py

```diff
@@ -295,7 +296,7 @@ def genconf(
         """
         test_config = {
             'cluster_name': 'SSH Installed DC/OS',
```

Comment:
Because our upgrade doc says to serve bootstrap files from the bootstrap host, and this upgrade procedure needs to match that doc. I assume that's our recommendation because `scp`ing files to each host is more tedious than running an nginx container and changing the bootstrap URL.

If the installer provided a command that only distributed bootstrap files among the cluster, we could use that here. But right now that's coupled to `--deploy`.